approach.tex

```diff
@@ -198,25 +144,26 @@ \subsubsection{Requirements and Test Objects}
 
 \subsubsection{Test Documents}
 
-Despite all information is contained in MagicDraw and Jira, it is important to have this baselined as appropriate. This is already happening for the requirements that are captured in requirements documents, and when approved issued into Docushare.
-The same shall happen for the test information, that live dynamically in jira.
+Though all tests information is contained in MagicDraw and Jira, it is important to have a  baselined test documentation in Docushare. .
+%This is already happening for the requirements that are captured in requirements documents, and when approved issued into Docushare.
 
-The {\bf Test Specification} document consist on a collections of test cases that covers a specific \product{} product as defined in the document tree. These collections of test cases are available in in Jira as views in the ``LSST Verification and Validation'' project, organized in folders.
+The {\bf Test Specification} document consists of a collections of test cases that cover a specific \product{} product as defined in the document tree and in \secref{sect:components}.
+These collections of test cases are available in in Jira as views in the ``LSST Verification and Validation'' project, and are organized in folders.
 
-Following are the sections that characterize a test specification
+The test specification sections are organized as follows:
 
 \begin{itemize}
-\item Introduction: includes subsections ``Objective'' and ``Scope'' to be edit and maintain manually by the author, not in Jira, and uploaded it in github.
-\item Approach: to edit and maintain manually by the author, not in Jira, and uploaded it in github. The relevant subsections are: ``Tasks and criteria'', `` Features to be tested'', ``Features not to be tested'', ``Pass/fail criteria'', ``Suspension criteria'', ``Naming convention''.
+\item Introduction: includes subsections ``Objective'' and ``Scope'' to edit and maintain manually by the author, not in Jira, and upload changes it in github.
+\item Approach: to edit and maintain manually by the author, not in Jira, and upload changes it in github. The relevant subsections are: ``Tasks and criteria'', `` Features to be tested'', ``Features not to be tested'', ``Pass/fail criteria'', ``Suspension criteria'', ``Naming convention''.
 \item Test Case Summary: this is a summary table extracted directly from Jira that will summarize all test cases included in the test specification. Do not edit it manually, all changes need to be done in Jira.
 \item Test Case Specification: this section will include all test cases details extracted automatically from Jira. Do not edit it manually, al changes need to be done in Jira.
 \end{itemize}
 
-One additional section or a first appendix should be added in order to provide proper document location on the verification elements.
+One additional section or an appendix should be added to provide pointers to documentation  on the verification elements.
```

Comment:
The verification elements are key components in the verification strategy introduced with the tooling, but they are not baselined in any document at the moment. The proposal is to have them baselined in the test spec, in appendix, or in  a specific section.